htdocs/comm/propal.php

```diff
@@ -117,7 +116,6 @@
 
 if (empty($reshook))
 {
-	if ($cancel) $action = '';
```

Comment:
Can you keep this.
For future, we must change code to follow this rule that is much secured than adding a test && ! GETPOST('cancel') on each following test